doc/en/components/layouts/color-editor.md

```diff
@@ -0,0 +1,179 @@
+---
+title: {Platform} Color Editor | Color Editor | Infragistics
+_description: Color Editor component provides an easily configurable option to change colors for any desirable component or aspect of your application.
+_keywords: {Platform} Color Editor, {ProductName}, Infragistics
+mentionedTypes: ["ColorEditor"]
+namespace: Infragistics.Controls
+---
+
+# {Platform} Color Editor Overview (preview)
+The {ProductName} Color Editor is a lightweight color picker component. The Color Editor can be clicked to pop open and modified. Both the rgba and hex values can be obtained from the desired color along the bottom. These values will update when the three sliders are modified. The center box is designed for adjusting the saturation and brightness. There two adjacent sliders for adjusting the rgb and luminance. Rgb registers between (1-255). The lightness registers between(0-1).
```

Comment:
A couple of typos to fix here.

- The Color Editor can be clicked to pop open and modified.
- There two adjacent sliders for adjusting the rgb and luminance.